support-frontend/app/controllers/DigitalSubscriptionController.scala

```diff
@@ -35,38 +38,47 @@ class DigitalSubscriptionController(
     if (orderIsAGift) "subscribe/digital/gift" else "subscribe/digital"
   )
 
-  def digital(countryCode: String, orderIsAGift: Boolean): Action[AnyContent] = CachedAction() { implicit request =>
-    implicit val settings: AllSettings = settingsProvider.getAllSettings()
+  def digital(countryCode: String, orderIsAGift: Boolean): Action[AnyContent] = {
+    maybeAuthenticatedAction() {
```

Comment:
All that has changed in this file is that everything has been indented because of the `maybeAuthenticatedAction`

Reply:
if you view with this link, the whitespace is ignored, so it's easier to review https://github.com/guardian/support-frontend/pull/3249/files?w=1